templates/freeradius/sqlcounter.j2

```diff
@@ -0,0 +1,65 @@
+# The dailycounter is included by default in the freeradius conf
+
+sqlcounter dailycounter {
+    sql_module_instance = sql
+    dialect = ${modules.sql.dialect}
+
+    counter_name = Daily-Session-Time
+    check_name = Max-Daily-Session
+    reply_name = Session-Timeout
+
+    key = User-Name
+    reset = daily
+
+    $INCLUDE ${modconfdir}/sql/counter/${dialect}/${.:instance}.conf
+}
+
+# The dailybandwidthcounter is added by OpenWISP
+sqlcounter dailybandwidthcounter {
+    sql_module_instance = sql
+    dialect = ${modules.sql.dialect}
+
+    counter_name = Max-Daily-Session-Traffic
+    check_name = Max-Daily-Session-Traffic
+    key = User-Name
+    reset = daily
+
+    $INCLUDE ${modconfdir}/sql/counter/${dialect}/${.:instance}.conf
+}
+
+sqlcounter noresetcounter {
+    sql_module_instance = sql
+    dialect = ${modules.sql.dialect}
+
+    counter_name = Max-All-Session-Time
+    check_name = Max-All-Session
+    key = User-Name
+    reset = never
+
+    $INCLUDE ${modconfdir}/sql/counter/${dialect}/${.:instance}.conf
+}
+
+sqlcounter monthlycounter {
+    sql_module_instance = sql
+    dialect = ${modules.sql.dialect}
```

Comment:
New problem, tests failing because ` ${modules.sql.dialect}` is not found.
I tried `dialect = "{{ freeradius_sql.dialect }}"` and that worked. Not sure why the value isn't found in `sqlcounter` now but should I make this change?